editor/editor_plugin.cpp

```diff
@@ -234,8 +234,8 @@ String EditorInterface::get_current_path() const {
 	return EditorNode::get_singleton()->get_filesystem_dock()->get_current_path();
 }
 
-void EditorInterface::inspect_object(Object *p_obj, const String &p_for_property) {
-	EditorNode::get_singleton()->push_item(p_obj, p_for_property);
+void EditorInterface::inspect_object(Object *p_obj, const String &p_for_property, bool inspector_only) {
```

Comment:
Parameters should be prefixed with `p_`.

Reply:
Fixed the parameter name. Sorry, still new to the format Godot uses!